fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -454,3 +454,62 @@ countries:
                 length: 3.0
                 issue_months: [8]
                 year_range: [1996, 2021]
+
+    djibouti:
+        logo: Djibouti_IRI.svg
+        resolution: [.25, .25]
+        center: [43, 12]
+        zoom: 8
+        rotation: 0
+        marker:  [43, 12]
+        origin: [.125, 125]
+        shapes:
+            - name: National
+              sql: select adm0_code as key, adm0_name as label, ST_AsBinary(the_geom) as the_geom
+                  from iridb.g2015_2012_0 where adm0_name = 'Djibouti'
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Djibouti' group by 1, 2
+
+            - name: Regional
+              sql: select code as key, djibouti_adm01 as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.djibouti_adm01
+              vuln_sql: select code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Djibouti' group by 1, 2
+
+            - name: District
```

Comment:
Should this be renamed Sub-region?

Reply:
I'm honestly not sure. Liban seemed to suggest to me that Subregion is correct but was inconsistent there.